treeherder/webapp/api/performance_data.py

```diff
@@ -707,10 +709,20 @@ def list(self, request):
         else:
             grouped_values = defaultdict(list)
             grouped_job_ids = defaultdict(list)
-            for signature_id, value, job_id in data.values_list('signature_id', 'value', 'job_id'):
-                if value is not None:
-                    grouped_values[signature_id].append(value)
-                    grouped_job_ids[signature_id].append(job_id)
+            if replicates:
+                for signature_id, value, job_id, replicate_value in data.values_list(
+                    'signature_id', 'value', 'job_id', 'performancedatumreplicate__value'
+                ):
+                    if replicate_value is not None:
+                        grouped_values[signature_id].append(replicate_value)
```

Comment:
@gmierz Could you provide a bit more context on why only the replicates values are needed in this case? I was expecting for the values to also be used along the replicates ones like so:
```suggestion
                        grouped_values[signature_id].append(replicate_value)
                        grouped_values[signature_id].append(value)
```

Reply:
Something I forgot to mention here is that I changed this to gather the value if the replicates is None. This would be for any performance datum that has no associated performance_datum_replicates entries. This should let us compare m-c commits to try commits with the replicates, and also look at metrics that don't have any replicates (like the sp2 score).